Editor/CoreAPI/Paths.cs

```diff
@@ -6,6 +6,7 @@ namespace AmazonGameLift.Editor
     public static class Paths
     {
         public const string PluginSettingsFile = "GameLiftSettings.yaml";
+        public const string PluginConfigurationFile = "GameLiftConfiguration.yaml";
```

Comment:
`PluginConfiguration` vs `PluginSettings` are identical names in terms of meaning. What is the difference between these two, and can we use that to come up with better naming?